tests/config.json

```diff
@@ -0,0 +1,8 @@
+{
+    "secret": "tests/dummysecret",
+    "issuer": "http://etos-api.com/token",
+    "authorization_backend": {
+        "name": "NoAuthentication",
```

Comment:
We should documents somewhere that this is case sensitive, since it is linked directly to the name of the class.

Reply:
Yes, I made a comment in this PR that the documentation would receive an update.